Home.md

```diff
@@ -1,40 +1,51 @@
+Start watching these two ([#1](http://prezi.com/0zh_ypsezu6i/), [#2](http://www.slideshare.net/JensMittelbach/dswarm-a-library-data-management-platform-based-on-a-linked-open-data-approach)) presentations, which summarize the motivation and goals on an abstract level.
 
-### d:swarm is an ETL tool
+## What can I do with d:swarm today?
 
-With the d:swarm ETL tool you can basically complete the following workflow: 
- * how to setup and configure [[data resources|Glossary#data-resource]]
- * create [[projects|Glossary#project]], to define [[mappings|Glossary#mapping]], [[transformations|Glossary#transformation]] and [[filters|Glossary#filter]]
- * explore the [[graph data model|Glossary#graph-data-model]]
- * and export data in [RDF](http://en.wikipedia.org/wiki/Resource_Description_Framework) or XML (e.g. for feeding [Solr](http://lucene.apache.org/solr/) indices).
+With the current d:swarm implementation you can ...
+ * import, set up and configure [[data resources|Glossary#data-resource]]
+ * create [[projects|Glossary#project]], define [[mappings|Glossary#mapping]], [[transformations|Glossary#transformation]] and [[filters|Glossary#filter]]
+ * transform data
+ * export data in [RDF](http://en.wikipedia.org/wiki/Resource_Description_Framework) or XML (e.g., for feeding [Solr](http://lucene.apache.org/solr/) indices).
 
-<!--- update image at https://intranet.slub-dresden.de/display/DAT/Technical+documentation+d%3Aswarm -->
-[[img/simplified-linear-dmp-steps.png]]
+<!--- source code for image currently stored by jan -->
+[[img/dswarm-workflow-abstract.png]]
+
+Configuring resources and creating mapping projects can be done with the [d:swarm BackOffice](http://demo.dswarm.org) web application. See our [user guide](https://github.com/dswarm/dswarm-documentation/wiki/Overview) for a brief manual of how to utilise the d:swarm BackOffice. While you can transform example data directly with the BackOffice, batch-processing large amounts of data can be done with the *[Task Processing Unit](https://github.com/dswarm/task-processing-unit-for-dswarm)* for d:swarm (TPU), initially developed by [UB Dortmund](https://www.ub.uni-dortmund.de/). When processing data with D:SWARM, you have the choice between two options - the *Streaming* and the *DataHub* version.
+
+### d:swarm *Streaming* Version
+
+The d:swarm *streaming* version offers fast processing of large amounts data and is sufficient for many scenarios. You may already use it today to start with as long as the work on the full *DataHub* version of D:SWARM continues. Unlike in the *DataHub* version this does not allow for versioning/archiving. See how  [[SLUB Dresden employs d:swarm | D:SWARM Use Case at SLUB Dresden]] for transforming and integrating bibliographic data sources. (Currently, streaming the export is implemented for XML, RDF will be added on demand.) 

```

Comment:
"You may already use it today to start with as long as the work on the full *DataHub* version of D:SWARM continues." - no, both variants are equal, i.e., the streaming variant is not just there, because of drawbacks of the data hub variant. the streaming variant is simply another use case (and offers other capacities)

-> "You can already utilise it today ..."